SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -79,7 +57,7 @@ void USpatialLatencyTracer::RegisterProject(UObject* WorldContextObject, const F
 bool USpatialLatencyTracer::SetTraceMetadata(UObject* WorldContextObject, const FString& NewTraceMetadata)
 {
 #if TRACE_LIB_ACTIVE
-	if (USpatialLatencyTracer* Tracer = GetTracer(WorldContextObject))
+	if (SpatialGDK::TracerSharedPtr Tracer = GetTracer(WorldContextObject))
```

Comment:
Things like this presumably don't need ownership.

Reply:
Yep, doesn't seem like it hurts though. If you prefer I can make these weak.

Reply:
It has a perfroamnce cost. You are doing an atomic increment on each call. + It makes the reader do a double take as they wonder why this is given ownership.

Reply:
WeakPtr have to be tracked also - but yeah like I said I can make that change.